src/components/RadioButtonGroup/readme.md

```diff
@@ -499,3 +486,69 @@ const Contacts = ({ data: contacts }) => {
         <Contacts data={data} />
     </div>
 ```
+
+
+##### RadioButtonGroup with different Border Radius
+
+```js
+
+    import React, { useState } from 'react';
+    import { RadioButtonGroup } from 'react-rainbow-components';
+
+    const options = [
+        { value: 'off', label: 'Off' },
+        { value: 'parking', label: 'Parking' },
+        { value: 'auto', label: 'Auto' },
+        { value: 'on', label: 'On' },
+    ];
+
+    const BorderRadiusRadioButtonGroup = () => {
+        const [value1, setValue1] = useState('auto');
+        const [value2, setValue2] = useState('auto');
+        const [value3, setValue3] = useState('auto');
+
+        const handleOnChange1 = event => {
+            setValue1(event.target.value);
+        }
+        const handleOnChange2 = event => {
+            setValue2(event.target.value);
+        }
+        const handleOnChange3 = event => {
+            setValue3(event.target.value);
+        }
+
+        return (
+            <>
+                <RadioButtonGroup
+                    label="Border Radius Square"
+                    id="radio-button-group-component-1"
+                    options={options}
+                    value={value1}
+                    onChange={handleOnChange1}
+                    borderRadius="square"
+                />
+                <RadioButtonGroup
+                    label="Border Radius Semi Rounded"
+                    id="radio-button-group-component-1"
+                    options={options}
+                    value={value2}
+                    onChange={handleOnChange2}
+                    borderRadius="semi-rounded"
+                />
+                <RadioButtonGroup
+                    label="Border Radius Rounded"
+                    id="radio-button-group-component-1"
```

Comment:
id here is not needed but if you will use id then they must be unique in all the page